populator-machinery/controller.go

```diff
@@ -975,7 +982,7 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 	return nil
 }
 
-func makePopulatePodSpec(pvcPrimeName string) corev1.PodSpec {
+func MakePopulatePodSpec(pvcPrimeName string) corev1.PodSpec {
```

Comment:
Does this signature need to change? I think the only callers are in this package.

Reply:
This also used in test